pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -72,6 +74,10 @@ type ResourceBindingSpec struct {
 	// +optional
 	Clusters []TargetCluster `json:"clusters,omitempty"`
 
+	// Placement represents the rule for select clusters to propagate resources.
+	// +optional
+	Placement policyv1alpha1.Placement `json:"placement,omitempty"`
```

Comment:
```suggestion
	Placement *policyv1alpha1.Placement `json:"placement,omitempty"`
```
We should use a pointer for optional fields.

For history reasons, PP violates the rule which will be fixed in v1alph2 or beta.